superset/sql_lab.py

```diff
@@ -98,6 +98,7 @@ def get_sql_results(
         query.status = QueryStatus.FAILED
         query.tmp_table_name = None
         sesh.commit()
+        raise
```

Comment:
If it's not passed to raise it's the one in the scope. I've verified it with both python3 and python2.